src/ingestor.ts

```diff
@@ -58,32 +58,30 @@ export default async function ingestor(req) {
     if (!Object.keys(hashTypes).includes(hash)) return Promise.reject('wrong types');
     type = hashTypes[hash];
 
-    network = '1';
-    let aliased = false;
     if (!['settings', 'alias', 'profile'].includes(type)) {
       if (!message.space) return Promise.reject('unknown space');
       const space = await getSpace(message.space);
       if (!space) return Promise.reject('unknown space');
       network = space.network;
-      if (space.voting?.aliased) aliased = true;
     }
 
     // Check if signing address is an alias
-    const aliasTypes = ['follow', 'unfollow', 'subscribe', 'unsubscribe', 'profile'];
-    const aliasOptionTypes = [
+    const aliasTypes = [
+      'follow',
+      'unfollow',
+      'subscribe',
+      'unsubscribe',
+      'profile',
       'vote',
       'vote-array',
       'vote-string',
       'proposal',
+      'update-proposal',
       'delete-proposal',
       'statement'
     ];
```

Comment:
what action types are not this list except `settings` and `alias`? maybe we can remove this list and check for `settings` and `alias` types only?